contracts/migration/upgradeable/AlphaKlimaRedeemUpgradeable.sol

```diff
@@ -56,7 +56,9 @@ contract AlphaKlimaRedeemUpgradeable is ERC2771ContextUpgradeable  {
 
         KLIMA.transfer(_msgSender(), amount);
     }
-    function setTrustedForwarder(address newForwarder) public onlyOwner {
-        _trustedForwarder = newForwarder;
-    }
+
+    // Can't do this https://ethereum.stackexchange.com/a/79604
```

Comment:
The README file has been updated to document the workaround that was used to get this contract originally deployed.

https://github.com/KlimaDAO/klimadao-solidity#build

We either keep the workaround documented, or remove the workaround and this function.